### PR TITLE
Fix load error

### PR DESCRIPTION
**Pull request by Konboi**
## Context

While I was implementing this plugin, a plugin loading error occurred.

image

First, I tried to update v2.361.4 because Pipeline API required it

https://gyazo.konboi.com/e588d921ce117e8781b7763e22b5efca.png

However, the error still occurred. So, I updated to v2.387.3.

image
image
image

Unfortunately, error wasn't resolved

image
image

Finally, I updated to the Jenkins version to v2.401.3 and loading error was resolved.

## Ask

- Could you try to reproduce this issue on your local environment? If you cannot reproduce this issue, I'll fix my local env.
- Honestly, I'm not sure these version combinations and what should we choose the Jenkins version. Could you review around that carefully?

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -26,11 +26,24 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.8.1</version>
         <configuration>
-          <source>1.8</source>
-          <target>1.8</target>
+          <source>11</source>
+          <target>11</target>
```
Comment (Konboi) [inline pom.xml:32]: Can we use Java v11?

File: pom.xml [hunk 2]
```diff
@@ -16,8 +16,8 @@
     <revision>1.0</revision>
     <changelist>-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
-    <jenkins.version>2.222.4</jenkins.version>
```
Comment (kohsuke) [inline pom.xml:19]: We intentionally depend on older version of Jenkins in order to maintain compatible with a wide range of Jenkins. There's nothing in this plugin that requires new version of Jenkins, so it's important to stick to the current base Jenkins version.

**Review by kohsuke — CHANGES_REQUESTED** [review]

**Comment by kohsuke** [comment]
Jenkins store everything in a directory, called "$JENKINS_HOME", including all the plugins installed. When you are developing a plugin like this, $JENKINS_HOME is `./work`.

When I start Jenkins clean with empty home, I do not get any errors, which suggests you probably have accidentally updated some plugin versions. Without the logs showing the plugin loading failure, it's hard to precisely pin down why, but probably there's no point.

I suggest you start by `rm -rf ./work/plugins` which just wipes the plugins, forcing the plugin dev environment to start with its default set of compatible plugins. I believe that should fix the problem.

**Comment by Konboi** [comment]
thank you, I'll try to do `rm -rf ./work/plugins`.

**Comment by Konboi** [comment]
Thank you, the original error was resolved!

**Pull request closed without merging**